src/utils/extractors.ts

```diff
@@ -0,0 +1,69 @@
+import { isArray } from 'lodash'
+import { IEndpointMiddleware, ISwaggerMiddlewareConfig } from '../types/interfaces'
+
+function extractMiddlewareName(configMiddleware: ISwaggerMiddlewareConfig): string {
+	const tempName = configMiddleware.middlewareName.replace('<', '').replace('>', '')
+
+	return `${tempName.charAt(0).toUpperCase() + tempName.slice(1).toLowerCase()}`
+}
+
+/**
+ * Works only with first middleware argument and prints his value if it consists of a primitive value array or an object containing such arrays
+ * @param endpointMiddleware object containing all necessary information about actual middleware
+ * or undefined in case the middleware is not used in a given endpoint.
+ * @param configMiddleware config information of a middleware wich should be processed
+ * @return { string } middleware's description
+ * */
+export const firstVersionExtractor = (endpointMiddleware: IEndpointMiddleware | undefined, configMiddleware: ISwaggerMiddlewareConfig): string => {
```

Comment:
@kot-1999 what does it mean "firstVersion" ? :D

Reply:
@LubomirIgonda1 Yeah, not the best naming. Would it be better if I call it basicArrayExtractor()/basicArrayFormatter() ?